### PR TITLE
Fixed SAIC-380: documentation and scenarios

### PR DESCRIPTION
**Pull request by realsystem**
Separated scenario for migration resources and VMs.

### CODE REVIEW

File: scenario/migrate_resources.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+namespace:
```
Comment (roman-verchikov) [inline scenario/migrate_resources.yaml]: Please add comments or docstring describing the purpose of this scenario and when to use it

File: scenario/migrate_vms.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
+namespace:
```
Comment (roman-verchikov) [inline scenario/migrate_vms.yaml]: Please add comments or docstring describing the purpose of this scenario and when to use it

**Comment by roman-verchikov** [comment]
Please fix inlines, also let's put a README file into `scenario` folder which describes how and when to use scenarios.

File: scenario/README [hunk 3]
```diff
@@ -0,0 +1,24 @@
+By default CloudFerry tool is going to migrate all resources and instances from the source cloud to the destination cloud.
```
Comment (roman-verchikov) [inline scenario/README]: The doc looks great, please modify it to use [markdown](http://daringfireball.net/projects/markdown/syntax).

File: scenario/migrate_vms.yaml [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+          - detach_volumes_on_source: False
+          - act_stop_vms: True
+          - transport_resource_inst:
+              - transport_images:
+                  - act_conv_comp_img: True
+                  - act_copy_inst_images: True
+                  - act_conv_image_comp: True
+              - set_volume_id_for_attaching: True
+          - transport_inst:
+              - act_net_prep: True
```
Comment (ntymtsiv) [inline scenario/migrate_vms.yaml]: after "act_net_prep" action should be action "associate_floatingip_on_dest"

File: scenario/README (hunk 3 above)
Comment (realsystem) [inline scenario/README]: example?

**Comment by roman-verchikov** [comment]
please rebase

File: scenario/README (hunk 3 above)
Comment (roman-verchikov) [inline scenario/README]: we have plenty in CF, for example, our primary readme uses markdown: https://github.com/MirantisWorkloadMobility/CloudFerry/blob/master/README.md

**Comment by roman-verchikov** [comment]
Please rebase and fix inlines

**Comment by roman-verchikov** [comment]
please rebase

**Comment by roman-verchikov** [comment]
@realsystem please rebase

**Comment by roman-verchikov** [comment]
+1, can be merged

**Pull request merged**